node_modules/browser/theme.js

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define((require, exports, module) => {
+
+// Note that all functions of `url` need to be called as methods.
+const url = require("./util/url.js");
+
+'use strict';
+
+const defaultTheme = {
+  glyphsShowing: false,
+  close: "#FC5753",
+  min: "#FDBC40",
+  max: "#33C748",
+  progress: "rgba(255, 255, 255, 0.4)",
+  location: "#eee",
+  buttons: "rgba(0, 0, 0, 0.5)",
+  domain: "rgba(0, 0, 0, 0.65)",
+  title: "rgba(0, 0, 0, 0.5)",
+  background: "#fff"
+};
+exports.defaultTheme = defaultTheme;
+
+const hardcodedColors = {
+  // [foreground, background]
```

Comment:
Good advice. Note that these booleans are localized to the file. These arrays are expanded into a more detailed "theme object" by `expandTheme`.